can-observation.js

```diff
@@ -101,6 +101,11 @@ function Observation(func, context, compute){
 	this.childDepths = {};
 	this.ignore = 0;
 	this.needsUpdate= false;
+	if (!func) {
+		this.start = function(){
```

Comment:
does this function need to be re-generated? Could only one be created and just use a variable?